src/Framework/BuildErrorEventArgs.cs

```diff
@@ -264,6 +315,15 @@ internal override void CreateFromStream(BinaryReader reader, int version)
             columnNumber = reader.ReadInt32();
             endLineNumber = reader.ReadInt32();
             endColumnNumber = reader.ReadInt32();
+
+            if (version >= 40)
+            {
+                helpLink = reader.ReadByte() == 0 ? null : reader.ReadString();
```

Comment:
nit:
Any reason not to crunch the conditionals together? Like:
`helpLink = version >= 40 && reader.ReadByte() != 0 ? reader.ReadString() : null;`
or
```
if (version >= 40 && reader.ReadByte() != 0)
{
    helpLink = reader.ReadString();
}
else
{
    helpLink = null;
}
```

? Same below.

Reply:
Not really, just tried to keep it in line with a similar construct a little higher up for the projectFile field. I don't mind changing it though, but I figured keeping in the same style was more important ;).